stratum/hal/lib/common/config_monitoring_service.cc

```diff
@@ -131,6 +134,39 @@ ::util::Status ConfigMonitoringService::PushChassisConfig(
   return ::util::OkStatus();
 }
 
+namespace {
+// Helper function to determine whether a container of protobuf messages have an
+// unique name field.
+template <typename T>
+bool AreUniqueNames(const T& values) {
```

Comment:
`HaveUniqueNames`

Reply:
Maybe "The collection **`contains`** unique names?"